packages/yarnpkg-pnpify/tests/NodePathResolver.test.ts

```diff
@@ -81,7 +86,13 @@ describe('NodePathResolver', () => {
   it('should resolve /home/user/proj/node_modules path', () => {
     const nodePath = NodeFS.toPortablePath('/home/user/proj/node_modules');
     const pnpPath = resolver.resolvePath(nodePath);
-    expect(pnpPath).toEqual({resolvedPath: '/home/user/proj/node_modules', statPath: '/home/user/proj', dirList: new Set(['foo', 'bar', '@scope'])});
+    expect(pnpPath).toEqual({resolvedPath: '/home/user/proj/node_modules', statPath: '/home/user/proj', dirList: new Set(['monorepo', 'foo', 'bar', '@scope'])});
+  });
+
+  it('should resolve /home/user/proj/node_modules/monorepo path', () => {
```

Comment:
I think with current implementation, there will be a problem with paths like:
`/home/user/proj/node_modules/monorepo/sources/index.ts`
It will mark this path as `isSymlink: true`. I believe this unit test should be added and the code adjusted if this problem indeed exists.

Reply:
Good point! Fixed 👍

Reply:
Thanks, looks good now!